src/Stratis.Bitcoin.Features.Consensus/ConsensusSettings.cs

```diff
@@ -58,20 +58,27 @@ private ConsensusSettings LoadFromConfig()
         }
 
         /// <summary>
-        /// Gets the default setting for the hash for the block to assume is valid.
+        /// Gets the default setting for the block hash of the block to assume is valid.
```

Comment:
Do we have a way to switch off AssumeValid?
I am not even sure we need to switch it off as if the block is not present it will default to validate it

Reply:
assumevalid=0 will switch it off, it's the same as core.